### PR TITLE
Ami auto update imprvmnts doc 2003

### PR DESCRIPTION
**Pull request by rickshar**
_This is the default pull request template. You can customize it by adding a `pull_request_template.md` at the root of your repo or inside the `.github` folder._

# Jira Ticket

_Include a link to your Jira Ticket_
_Example: [JIRAISS-1234](https://spotinst.atlassian.net/browse/JIRAISS-1234)_

# Demo

_Please add a recording of the feature/bug fix in work. if you added new routes, the recording should show the request and response for each new/changed route_

# Checklist:
- [ ] I have filled relevant self assessment ([NodeJS](https://docs.google.com/forms/d/e/1FAIpQLSfl14u9AOBAmxVJ272tvO7XNuXE-EMvWaGcaRZalu1UAKB7RA/viewform), [Frontend](https://docs.google.com/forms/d/e/1FAIpQLSdiBPNKH81w_EkavihVL8Uwb0j7tP8PwJmLFYm2nCOQxz-1qw/viewform), [Backend](https://docs.google.com/forms/d/e/1FAIpQLSed_PsTJ5-XIWkFL6BSDE2AQRBVPmwc3PAmHZkUn-erzVI37Q/viewform?usp=sf_link))
- [ ] I have run ESlint on my changes and fixed all warnings and errors (**NodeJS & Frontend Services**)
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have validated all the requirements in the Jira task were answered
- [ ] I have all neccessary approvals for the design/mini design of this task
- [ ] I have approved the API changes and granular permission patterns (documentation subtask) (**For public services only**)

### CODE REVIEW

File: src/docs/ocean/features/ami-auto-update-eks-ui.md [hunk 1]
```diff
@@ -21,30 +21,33 @@ If you have not run or scheduled an auto-update in this cluster, the AMI Auto Up
...
-  * Success (green): The update was successful.
-  * No update required (blue): Hover over the status icon for more information.
-  * Error (red): Error at Virtual Node Group level: The update has failed for this specific VNG. Hover over the error icon for more information.
-  * Cluster Error (red): The cluster update has failed. Hover over the cluster error icon for more information.
+* Trigger Type: Manual (auto-update was run immediately) or Scheduled (auto-update was scheduled and run).
+* Status (per batch):
+  * Completed (green): All Virtual Node Groups in the batch were updated successfully.
+  * Partially completed (orange): Either at least one Virtual Node Group did not require updates or the update failed.
+  * No update required (blue): No Virtual Node Groups required update.
+  * Failed (red): The batch update failed. Hover over the error icon for more information.
```
Comment (NofarNak) [inline src/docs/ocean/features/ami-auto-update-eks-ui.md:34]: @rickshar  The Failed status can be displayed in 2 cases - where the whole update has failed, then there will be a tooltip in the icon with the issue OR if all the VNGs failed to update and the error will be inside the table for each VNG.

File: src/docs/ocean/features/ami-auto-update-eks-ui.md [hunk 2]
```diff
@@ -21,30 +21,33 @@ If you have not run or scheduled an auto-update in this cluster, the AMI Auto Up
...
+ * Status (per cluster):
+  * Failed (red - with no down arrow for Virtual Node Groups): Cluster failed. Hover over the error icon for more information.
```
Comment (NofarNak) [inline src/docs/ocean/features/ami-auto-update-eks-ui.md:36]: @rickshar The status is per VNG not cluster, and there are 3 statuses - Completed, Failed, No Update Required

File: src/docs/ocean/features/ami-auto-update-eks-ui.md (hunk 1 above)
Comment (NofarNak) [inline src/docs/ocean/features/ami-auto-update-eks-ui.md:34]: @rickshar Here, they don't have a tooltip for hovering.

File: src/docs/ocean/features/ami-auto-update-eks-ui.md [hunk 3]
```diff
@@ -21,30 +21,33 @@ If you have not run or scheduled an auto-update in this cluster, the AMI Auto Up
...
-* Roll ID: Listed if the cluster was rolled after auto-update. Click the link to view Roll ID attributes.
-* Trigger Type: Manual (auto-update was run immediately) or Scheduled (auto-update was scheduled and run).
-
+* Type: Security patches, Kubernetes minor version, or both.
+* Old AMI/New AMI: version numbers before and after the run.
+* Roll ID: This is listed if the cluster was rolled after the auto-update. Click the link to view Roll ID attributes.
+* Status:
+  * Completed (green): The Virtual Node Group update was successful.
+  * No update required (blue): The Virtual Node Group did not require an update. Hover over the status icon for more information.
+  * Error (red): Update failed at Virtual Node Group level: Hover over the error icon for more information.
```
Comment (NofarNak) [inline src/docs/ocean/features/ami-auto-update-eks-ui.md:47]: @rickshar Failed, not Error

File: src/docs/ocean/features/ami-auto-update-eks-ui.md [hunk 4]
```diff
@@ -21,30 +21,33 @@ If you have not run or scheduled an auto-update in this cluster, the AMI Auto Up
...
 
 >**Tip**: Use the Updates History filter to search for auto-update runs by Virtual Node Groups, Status, or Trigger Type.
```
Comment (NofarNak) [inline src/docs/ocean/features/ami-auto-update-eks-ui.md:49]: @rickshar Remove the Virtual Node Groups

File: src/docs/ocean/features/ami-auto-update-eks-ui.md [hunk 5]
```diff
@@ -21,30 +21,31 @@ If you have not run or scheduled an auto-update in this cluster, the AMI Auto Up
...
+  * Partially completed (orange): Either at least one Virtual Node Group did not require updates or the update failed.
+  * No update required (blue): No Virtual Node Groups required update.
+  * Failed (red): Either the whole update has failed (click the tooltip on the icon for more information), or all the Virtual Node Groups could not be updated (see the issue description in the list for the specific Virtual Node Group).
 
+Click on the down arrow for a batch to view information **per Virtual Node Group**:
 
+* Virtual Node Group: Click the link to view the Virtual Node Group attributes.
+* Type: Security patches, Kubernetes minor version, or both.
+* Old AMI/New AMI: Version numbers before and after the run.
+* Roll ID: This is listed if the cluster was rolled after the auto-update. Click the link to view Roll ID attributes.
```
Comment (NofarNak) [inline src/docs/ocean/features/ami-auto-update-eks-ui.md:41]: @rickshar 'This is listed if the VNG...' not the cluster

File: src/docs/ocean/features/ami-auto-update-eks-ui.md [hunk 6]
```diff
@@ -21,30 +21,31 @@ If you have not run or scheduled an auto-update in this cluster, the AMI Auto Up
...
+* Status:
+  * Completed (green): The Virtual Node Group update was successful.
+  * No update required (blue): The Virtual Node Group did not require an update. Hover over the status icon for more information.
+  * Failed (red): Update failed at Virtual Node Group level.
```
Comment (NofarNak) [inline src/docs/ocean/features/ami-auto-update-eks-ui.md:45]: @rickshar please add "Hover over the status icon for more information"

File: src/docs/ocean/features/ami-auto-update-eks-ui.md (hunk 4 above)
Comment (NofarNak) [reply src/docs/ocean/features/ami-auto-update-eks-ui.md:47]: @rickshar You didn't change it

File: src/docs/ocean/features/ami-auto-update-eks-ui.md (hunk 6 above)
Comment (rickshar) [reply src/docs/ocean/features/ami-auto-update-eks-ui.md:45]: @NofarNak please add the line number so I know where to implement this change.

**Pull request merged**